pydra/engine/submitter.py

```diff
@@ -16,24 +17,31 @@
 class Submitter:
     """Send a task to the execution backend."""
 
-    def __init__(self, plugin="cf", **kwargs):
+    def __init__(self, plugin: ty.Union[str, Worker] = "cf", **kwargs):
         """
         Initialize task submission.
 
         Parameters
         ----------
-        plugin : :obj:`str`
-            The identifier of the execution backend.
+        plugin : :obj:`str` or :obj:`pydra.engine.core.Worker`
+            Either the identifier of the execution backend or the backend itself.
             Default is ``cf`` (Concurrent Futures).
 
```

Comment:
Should document the kwargs.

```suggestion
        \*\*kwargs ...
```

Reply:
done